plugins/auto-sizes/hooks.php

```diff
@@ -41,4 +41,4 @@ function auto_sizes_render_generator(): void {
 add_filter( 'render_block_core/image', 'auto_sizes_filter_image_tag', 10, 3 );
 add_filter( 'render_block_core/cover', 'auto_sizes_filter_image_tag', 10, 3 );
```

Comment:
Not really related to this PR, but ideally we also need to support the `core/post-featured-image` block for this feature. Maybe we can open another PR to capture this task.

Reply:
Make sense. Added on overview issue #760